frontend/app/assets/css/main.css

```diff
@@ -53,6 +53,16 @@
   background-clip: text;
 }
 
+@utility stroked-text-2 {
+  -webkit-text-stroke-color: transparent;
```

Comment:
I think you can use `@apply <other utility>` to reduce code duplciation, and pick better names